devsetup/scripts/tripleo.sh

```diff
@@ -101,14 +96,13 @@ export OCTAVIA_ENABLED=${OCTAVIA_ENABLED}
 export TELEMETRY_ENABLED=${TELEMETRY_ENABLED:-true}
 
 set +x
-if [[ -f \$HOME/containers-prepare-parameters.yaml ]]; then
-    echo "Using existing containers-prepare-parameters.yaml - contents:"
-    cat \$HOME/containers-prepare-parameters.yaml
-else
+if [ ! -f \$HOME/containers-prepare-parameters.yaml ]; then
     login_args=" "
     [ "\$RH_REGISTRY_USER" ] && [ -n "\$RH_REGISTRY_PWD" ] && login_args="--enable-registry-login"
+    set -x
     openstack tripleo container image prepare default \
         --output-env-file \$HOME/containers-prepare-parameters.yaml \${login_args}
+    set +x
     # Use wallaby el9 container images
     sed -i 's|quay.io/tripleowallaby$|quay.io/tripleowallabycentos9|' \$HOME/containers-prepare-parameters.yaml
```

Comment:
Since you're here and in the mood for improvements, you can also remove this sed since there is no more wallaby tripleo deploy here/